staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -117,7 +117,8 @@ type Config struct {
 	CorsAllowedOriginList []string
 	HSTSDirectives        []string
 	// FlowControl, if not nil, gives priority and fairness to request handling
-	FlowControl utilflowcontrol.Interface
+	FlowControl               utilflowcontrol.Interface
+	PriorityAndFairnessConfig utilflowcontrol.PriorityAndFairnessConfig
```

Comment:
I would probably probably call it FlowControlConfig for consistency.

Reply:
Done.